azerothcore/azerothcore-wotlk
# fix(DB/creature_template): Adjust levels for Snobold Vassal and Saronite Animus

## CHANGES PROPOSED:
-  Increase level of Snobold Vassal (Trial of the Crusader 10N/H & 25N/H) from 78 to 82 [See video for proof](https://youtu.be/po9k71NjnE4?t=123) - You can make out the level of the mob by the tooltip on the top of the video.  This will make the NPC match levels with adds summoned for bosses like Jaraxxus and Anub'arak, which are also 82 elites.
-  Increase level of Saronite Animus (Ulduar 10 & 25) to 83 so it mechanically acts like a boss, previously it was 76 on 10 man and 80 on 25 man (displays as skull)


## TESTS PERFORMED:
- Applied SQL
- Summoned the NPCs in question and checked level
- For Saronite Animus, `.npc info` to check level, as it is a Skull mob.


## HOW TO TEST THE CHANGES:
- Apply SQL, restart server, create character
- Run Trial of the Crusader on all 4 difficulties, pull the first boss, and check the level of the Snobold Vassal that jumps on you.
- Run Ulduar, get to General Vezax, activate hardmode and allow the Saronite Animus to spawn - use the command `.npc info` to check the level.
- Alternatively, summon the npcs in question with the following commands:
```
.npc add 33524
.npc add 34152
.npc add 34800
.npc add 35441
.npc add 35442
.npc add 35443

```



## Target branch(es):
